app/crawler/naver_map_crawler.py

```
@@ -127,6 +137,24 @@ def _extract_apollo_state_sync(self, page) -> List[Dict]:
             }
         """)
 
+    def _get_random_ua(self) -> str:
+        """Generates a random User-Agent string."""
+        try:
+            from fake_useragent import UserAgent
+            ua = UserAgent(platforms='pc', os='windows', browsers=['chrome', 'edge'])
+            return ua.random
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)